### ISSUE TITLE
Allow advertisers to see live opportunities

### ISSUE THREAD

**Issue by kimcashwell** [FMTCco/app-fresh.press#11]
From a new ticket this morning (hannahk@stellarbluetechnologies.com):

"I'm not sure if there is some technical issue, but all day I have been trying to go in to communicate with an influencer who bid on our opportunity and have been unable. It is showing I do not have any opportunities open, but I should have two currently."

Will update with screenshots when available.

**Comment by jamesvweston** [comment]
This is another one where I can't reproduce it.
Can you ask if they are still having the issue? What browser they're using? Have they tried to simply refresh the page? Have they tried logging out and logging back in?

**Comment by kimcashwell** [comment]
I reached out to her and asked. Will update if/when I hear back.

**Comment by kimcashwell** [comment]
This is the issue Chrystal is experiencing - I thought we had closed it but looks like it was a different one. Here's what she added:

>Able to see opportunities on Firefox, but not Google Chrome (I've heard similar from other people)
>I cleared the cache and everything is still gone. It's odd that even in the admin section I can see nothing on Google Chrome but I can in Firefox.
>Suggested you login under her FPM account (chrystal@fmtc.co) to see if you experience the same issue

**Comment by kimcashwell** [comment]
Received an email on 5/31 from kristine.kirschke@partnercentric.com who is experiencing this issue as well.

**Comment by kimcashwell** [comment]
It is now not displaying users, either. 
<img width="1154" alt="screen shot 2018-06-26 at 7 01 24 am" src="https://user-images.githubusercontent.com/24657284/41917340-fe8cdd52-790e-11e8-80cf-f6e2c7570d67.png">
<img width="1154" alt="screen shot 2018-06-26 at 7 01 46 am" src="https://user-images.githubusercontent.com/24657284/41917341-feae7b06-790e-11e8-80a9-1988080c87c7.png">

**Comment by melaniemay79** [comment]
Kim,

Can you try it from a non-admin POV? Try this URL:

https://app.freshpress.media/search_opportunities?order_by=opportunity.id&direction=desc&page=1&per_page=20&opportunity_status_ids=2

**Comment by kimcashwell** [comment]
Hmm, that link loads a blank page as well:

<img width="1153" alt="screen shot 2018-06-26 at 9 01 24 am" src="https://user-images.githubusercontent.com/24657284/41924769-bcce5128-791f-11e8-9eaf-cb371e3f2b58.png">


It's worth noting that influencers and advertisers are experiencing this issue too (where everything is blank) so I'm not sure it has to do with the admin POV.

**Comment by melaniemay79** [comment]
Does the search form work for you? Try using the deliverable type dropdown box to select something and search.

**Comment by kimcashwell** [comment]
No, I've tried several different searches and nothing shows up (both for users and opportunities).
